### DISCUSSION TITLE
Video Players to use with Slinger

### DISCUSSION THREAD

**Discussion by MikeVTO** [GerryDazoo/Slinger#391] [Q&A] [answered]
So I have VLC running on both my Windows PC and Android phone to watch the Slinger stream.  They work great.  However, on my iPad it runs terribly.   Freezes after about a minute.  Won’t cast to my Chromecast.

Is anyone using a different app on an iPad that streams Slinger well and is able to cast to a Chromecast?

Thanks for any suggestions! 😀

**Comment by DoogieHowitzerWMD** [comment, marked as answer]
Some folks have been experiencing issues with viewing the streams using VLC on Apple devices, and it is worse on iPhone and iPad particularly.

Per #208, there are other media players on Apple store that supposedly work better (OPlayer Lite HD and PlayerExtreme). However, you probably need to experiment with getting either one of those to cast with Chromecast.

**Reply by sp222222** [reply]
idk about casting but I changed my old old ipad VLC to highest latency setting deblock med, deinterlace off and hardware coding on. seems to finally be stabile. I had forgotten to use the thing at all when you mentioned iPAD. I am streaming atsc from proHD tuner. 16/2560/60/3 ini settings.

**Reply by DoogieHowitzerWMD** [reply]
I was trying to get the stream going with VLC on an iPhone (not mine, so I don't have it on hand), but I'll have to try changing VLC settings on it when I have a chance.

So, deinterlace off and hardware decoding on are pretty straightforward, and I'm assuming you used the network cache setting to increase the latency (if it was some other setting, please let me know). I'm not sure what deblock is though - you set that to medium or something?

**Reply by sp222222** [reply]
works better with medium on that... also I changed it to deinter ON, but must change latency setting to cache

**Reply by DoogieHowitzerWMD** [reply]
I'll give that a try....

also @sp222222, if you're up for helping someone, I see socelillyhammer in another thread (#117) is having trouble getting compatible remote keycodes with their cable box. 

I specifically thought that you might have an idea how someone would go about troubleshooting that, since you and Gerry worked through a lot of remote keycode issues months back.

**Reply by MikeVTO** [reply]
Thanks for these tips.  It is much more stable now.  I had to try a few things with VLC but the best so far is.

Video/ deblocking filter = medium
Video/deinterlace = on
Network/network caching level = high latency

**Comment by colfra** [comment]
I found on all my Apple devices that simply disabling hardware encoding in VLC's settings fixed the problem I had, namely the picture freezing after a few seconds. This was necessary and worked on my MacBook, an iPad, an iPad Pro and an iPhone. Interesting that it wasn't needed on Windows on my Macbook (under Parallels), but probably hardware decoding is not available there. PlayerXtreme and OPlayer Lite both worked without having to touch the settings.
